aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -231,6 +231,22 @@ CREATE_UNARY_UFUNC(tanh);
 CREATE_UNARY_UFUNC(trunc);
 CREATE_UNARY_UFUNC(conj_physical);
 
+// With addition of `round.decimals` overload, using CREATE_UNARY_UFUNC leads
+// to unresolved overload.
```

Comment:
Currently haven't added the `decimals` support for Sparse CSR
cc: @IvanYashchuk @krshrimali @mruberry

Reply:
Alternatively, the `CREATE_UNARY_UFUNC` macro could be modified to use `ATEN_FN` and `ATEN_FN2` to get unambiguous functions.

It's also fine to leave `round` separate as it is in this PR.

Reply:
Thanks @IvanYashchuk. Let's keep it seperate for now.